Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:16
```

```suggestion
FROM node:19
```

`package.json` requires v18, it seems to work also on v19
https://github.com/shobrook/adrenaline/blob/master/package.json#L56

---

Yep